.github/workflows/automationTemplate.yaml

```diff
@@ -16,7 +16,7 @@ on:
         description: 'The platform name should be all lowercase'
 
 env:
-  MAX_RETRIES: 10
+  MAX_RETRIES: 15
```

Comment:
I am wondering how increasing the number of retries will mitigate the issue?

Reply:
If the he pipeline does not finish within 1.5hrs then the pipeline will fail. Currently its taking more than that (shown in the picture of the description) takes over 2 hours.

Reply:
If the pipeline does not finish within 1.5hrs then the pipeline will fail. Currently its taking more than that (shown in the picture of the description) takes over 2 hours.